packages/typescript-estree/src/node-utils.ts

```diff
@@ -668,12 +669,13 @@ export function createError(
   message: string,
 ): TSError {
```

Comment:
`TSError` is still present, maybe we should just replace it with new error class?